src/containers/markdown/AddMarkdownFile.js

```diff
@@ -0,0 +1,44 @@
+import React, { PureComponent } from 'react';
+import { getMarkdownFiles } from '../../selectors/markdownSelectors';
+import store from '../../store';
+import AddButton from '../../components/markdown/AddButton';
+import { addMarkdownFile } from '../../actions/markdownActions';
+
+export default class AddMarkdown extends PureComponent {
+  state = {
+    markdownFiles: [],
+    unsubscribe: null
+  };
+
+  updateState = () => {
```

Comment:
Seems like this code is out of scope. This is code that would be used to display files